.gitlab/llnl-ci-conda.yml

```diff
@@ -0,0 +1,117 @@
+stages:
+  - env-setup
```

Comment:
let's have it just `setup`, since it includes (or high include) all preparation steps

Reply:
addressed it